tooling-tests/src/test/resources/test-src/input/tool_test_generate_1/Config.toml

```diff
@@ -0,0 +1,6 @@
+[foo.perist_generate_1]
```

Comment:
```suggestion
[foo.persist_generate_1]
```

Reply:
fix the typo in all places

Reply:
Updated in [d6a53d3](https://github.com/ballerina-platform/persist-tools/pull/15/commits/d6a53d3f948c8f076acb3cc1f2730258a998cf40)